extensions/wikia/Helios/User.class.php

```diff
@@ -2,24 +2,31 @@
 
 namespace Wikia\Helios;
 
+use Wikia\Logger\WikiaLogger;
+
 /**
  * A helper class for dealing with user-related objects.
  */
 class User {
 
-	const ACCESS_TOKEN_COOKIE = 'access_token';
+	const ACCESS_TOKEN_COOKIE_NAME = 'access_token';
+
+	// This is set to 6 months, 6*30*24*60*60 = 15552000
+	const ACCESS_TOKEN_COOKIE_TTL = 15552000;
```

Comment:
Huh, so that works out to ```365.243055555554``` days which doesn't match any of the possible year lengths given on Wikipedia: http://en.wikipedia.org/wiki/Year#Summary

I think they should be the same, but calculated so nobody has to look at a wikipedia page to figure out why the value is anywhere from 5 minutes to 2 days off expected.  I'll update the value here to be ```(365/2)*24*60*60``` and ask for it to be updated on Mercury to the same.